src/components/LandingPageHeader/index.js

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react'
+import {useHistory} from "react-router-dom";
+import clsx from 'clsx';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import useThemeContext from '@theme/hooks/useThemeContext';
+import './styles.css'
+
+
+function Action({title, link, description}) {
+    let [hovering, setHovering] = useState(false);
+    let history = useHistory();
+
+    const handleClick = (e) => {
+        e.preventDefault();
+        history.push(window.location.pathname + "/" + link);
```

Comment:
Maybe it is better to use [useBaseUrl](https://docusaurus.io/docs/docusaurus-core#usebaseurl) here, because it handles edge cases like root paths and links that already contain the base path too.

```suggestion
        history.push(useBaseUrl(link));
```